### PR TITLE
Added Chinese translation of documentation for drive policy training.

### PR DESCRIPTION
**Pull request by leegang**

### CODE REVIEW

File: body/README.md [hunk 1]
```diff
@@ -33,23 +33,23 @@ We were able to print the chassis with PLA, ABS and CPE. In our experience the p
...
-- 4x TT motors with tires ([EU](https://www.conrad.de/de/p/joy-it-com-motor01-getriebemotor-gelb-schwarz-passend-fuer-einplatinen-computer-arduino-banana-pi-cubieboard-raspbe-1573543.html), [US](https://www.amazon.com/dp/B07ZT619TD), [AE](https://www.aliexpress.com/item/4000126948489.html))
-- 1x L298N Motor Driver ([EU](https://www.conrad.de/de/p/joy-it-motormodul-2-u-4-phasen-6-bis-12v-1573541.html), [US](https://www.amazon.com/dp/B07ZT619TD), [AE](https://www.aliexpress.com/item/32994608743.html))
+- 4x TT motors with tires ([EU](https://www.conrad.de/de/p/joy-it-com-motor01-getriebemotor-gelb-schwarz-passend-fuer-einplatinen-computer-arduino-banana-pi-cubieboard-raspbe-1573543.html), [US](https://www.amazon.com/dp/B0818Y9G3N), [AE](https://s.click.aliexpress.com/e/_d9leRia))
+- 1x L298N Motor Driver ([EU](https://www.conrad.de/de/p/joy-it-motormodul-2-u-4-phasen-6-bis-12v-1573541.html), [US](https://www.amazon.com/dp/B0818Y9G3N), [AE](https://s.click.aliexpress.com/e/_dZaQCaI))
```
Comment (thias15) [inline body/README.md:37]: This updated Amazon link is not available.

File: body/README.md [hunk 2]
```diff
@@ -33,23 +33,23 @@ We were able to print the chassis with PLA, ABS and CPE. In our experience the p
...
+- 3x 18650 battery ([EU](https://www.conrad.de/de/p/conrad-energy-18650-usb-spezial-akku-18650-li-ion-3-7-v-1400-mah-1525536.html), [US](https://www.amazon.com/dp/B083K4XSKG), [AE](https://s.click.aliexpress.com/e/_dSw0T3y))
 - 1x 18650 battery holder([EU](https://www.amazon.de/dp/B075V25QJ9), [US](https://www.amazon.com/dp/B07DWQYD7H), [AE](https://www.aliexpress.com/item/33037738446.html))
-- 1x USB OTG cable ([EU](https://www.amazon.de/gp/product/B075M4CQHZ) ,[US](https://www.amazon.com/dp/B07LBHKTMM), [AE](https://www.aliexpress.com/item/10000330515850.html))
-- 1x spring or rubber band ([EU](https://www.amazon.de/gp/product/B01N30EAZO/), [US](https://www.amazon.com/dp/B008RFVWU2), [AE](https://www.aliexpress.com/item/33043769059.html))
+- 1x USB OTG cable ([EU](https://www.amazon.de/gp/product/B075M4CQHZ) ,[US](https://www.amazon.com/dp/B07LBHKTMM), [AE](https://s.click.aliexpress.com/e/_dWhjoMa))
+- 1x spring or rubber band ([EU](https://www.amazon.de/gp/product/B01N30EAZO/), [US](https://www.amazon.com/dp/B008RFVWU2), [AE](https://s.click.aliexpress.com/e/_dSINmUw))
```
Comment (thias15) [inline body/README.md:43]: This is the wrong spring. Spring with outer diameter 8 or 10 is needed.

**Review by thias15 — CHANGES_REQUESTED** [review]
Please check all the links you changed. Also git does not detect the previous changes, so the git history will shows more than 20 commits. Could you please do a fresh fork and only apply your changes in one or two commits for the PR. Thanks a lot!

**Comment by leegang** [comment]
> Please check all the links you changed. Also git does not detect the previous changes, so the git history will shows more than 20 commits. Could you please do a fresh fork and only apply your changes in one or two commits for the PR. Thanks a lot!

OK. Wait

**Pull request closed without merging**